src/code/z_play.c

```diff
@@ -504,16 +505,16 @@ void Play_Update(PlayState* this) {
             switch (this->transitionMode) {
                 case TRANS_MODE_SETUP:
                     if (this->transitionTrigger != TRANS_TRIGGER_END) {
-                        s16 sceneSetupIndex = 0;
+                        s16 sceneLayer = 0;
```

Comment:
child day layer?

(also bruh I didn't know the "continue bgm" flag was only checked on that layer's entrance entry)

Reply:
I saw this as just being an initilizer for the variable and not actually child day, but i think youre right. because of this, the continue bgm flag only matters when applied to the child day layer or a cutscene lol

I think thats probably not intentional? but that seems to be what the case is.
This probably needs a note in entrance_table.h or something